pyrene/src/components/Button/Button.tsx

```diff
@@ -76,6 +81,7 @@ const Button: React.FC<ButtonProps> = ({
       }
       onClick={onClick}
       disabled={disabled}
+      style={{ width: width }}
```

Comment:
this doesn't have a unit then, it's an error no? shouldn't it be 
```
style={{ width: `${width}px`}}
```
instead?

Reply:
Also, in this case you need to a check if `width` is undefined before as well

Reply:
If it is a `number` as specified by the type, React automatically adds the px, and if width is `undefined` then width is is not set.

Reply:
Ok, that works for me